sbt

```diff
@@ -1,132 +1,124 @@
 #!/usr/bin/env bash
 #
 # A more capable sbt runner, coincidentally also called sbt.
-# Author: Paul Phillips <paulp@typesafe.com>
```

Comment:
Are the changes in this file really mandatory for 7.2? If not, I just split them in another pr updating just the sbt script